autoload/ctrlsf/win.vim

```diff
@@ -119,6 +119,13 @@ func! s:InitMainWindow() abort
     setl textwidth=0
     setl nospell
     setl nofoldenable
+    if exists("*g:CtrlSFCustomizeWindowFunction")
+        if exists("b:ctrlsf_customized_window_function_called")
+            return
+        endif
+        let b:ctrlsf_customized_window_function_called = 1
+        silent! call g:CtrlSFCustomizeWindowFunction()
+    end
```

Comment:
I think this is better to be placed **right before** line 148 `let b:ctrlsf_initialized = 1`, let user can overwrite every aspect of default window initialization.